### PR TITLE
Add a package that enables the use of the volume buttons

### PR DESCRIPTION
**Pull request by prurigro**
This is an adaptation of the solution used by https://pureos.ironrobin.net/git/clover/pinephone to enable volume control with the volume buttons.

It includes:

1. The keyboard shortcut daemon actkbd
2. An actkbd config that enables volume control
3. A system service that sets the correct permission on the volume button device at boot time
4. A user service that starts actkbd when the user session starts
5. An install script that informs the user how to get up and running after the first install

Assuming you like this particular solution, let me know if you want me to make any modifications before you merge.

Cheers!

### CODE REVIEW

File: PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Maintainer: Danct12 <danct12@disroot.org>
+# Contributor: Kevin MacMartin <prurigro@gmail.com>
+# Adapted from https://pureos.ironrobin.net/git/clover/pinephone
+
+_pkgname=actkbd
+pkgname=$_pkgname-pinephone-volume
+pkgver=0.2.8
+pkgrel=1
```
Comment (Danct12) [inline PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD:8]: pkgrel should be 0 for new packages

File: PKGBUILDS/pine64/actkbd-pinephone-volume/pinephone-volume-permfix.service [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Make the user own the volume button device
+
+[Service]
+Type=oneshot
+ExecStart=chown 1000:1000 /dev/input/by-path/platform-1c21800.lradc-event
```
Comment (Danct12) [inline PKGBUILDS/pine64/actkbd-pinephone-volume/pinephone-volume-permfix.service:6]: I'm thinking there should be a way to merge this service with the actkbd service, right?

**Comment by Danct12** [comment]
Thank you for your contribution as always.

Anyway, this is a temporary solution for the volume button as this is not needed in the future. See [this merge request from upstream](https://source.puri.sm/Librem5/phosh/-/merge_requests/537).

File: PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD (hunk 1 above)
Comment (prurigro) [reply PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD:8]: Are you sure about that? I've never seen an arch package with a pkgrel of 0, and VCS packages are auto-reset to 1 when the version changes. Unless this is specific to your overlay?

File: PKGBUILDS/pine64/actkbd-pinephone-volume/pinephone-volume-permfix.service (hunk 2 above)
Comment (prurigro) [reply PKGBUILDS/pine64/actkbd-pinephone-volume/pinephone-volume-permfix.service:6]: I'd initially written them together, but it turned out that the actkbd service has to be started with --user in order to work. That means the service doesn't have the root permissions required to change the ownership on the device.

File: PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD (hunk 1 above)
Comment (Danct12) [reply PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD:8]: Oh, then I guess it's specific to me.

I've worked on Alpine Linux packages a lot, and they use `pkgrel=0` for their new packages.

Comment (prurigro) [reply PKGBUILDS/pine64/actkbd-pinephone-volume/PKGBUILD]: OK, I've updated the pkgrel to match the other packages in this overlay. I saw your other post though, so I guess it's up to you whether you want to offer this as a hold-over solution until upstream finishes fleshing stuff out or just leave this PR up until then so people can build it themselves until then.

**Comment by Danct12** [comment]
Deprecated. This is now done by: https://github.com/dreemurrs-embedded/Pine64-Arch/commit/2490502d08b58277ee03684f864a678cef7b5be4 and https://github.com/dreemurrs-embedded/Pine64-Arch/commit/1a15e4a2992072d8837cbf5137d7c0efb84eda1b

**Pull request closed without merging**